src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -367,9 +370,26 @@ gl_sframe style_transfer::predict(variant_type data,
       case flex_type_enum::VECTOR:
         style_idx = std::move(flex_style_idx.get<flex_vec>());
         break;
+      case flex_type_enum::LIST:
+      {
+        auto list = flex_style_idx.get<flex_list>();
+        style_idx.resize(list.size());
+        std::transform(list.begin(), list.end(),
+                   std::back_inserter(style_idx), [](flexible_type val) {
+                     return static_cast<double>(val.get<flex_float>());
+                   });
+        break;
+
+      }
+      case flex_type_enum::UNDEFINED: {
+        int num_styles = read_state<flex_int>("num_styles");
```

Comment:
The type of the variable should be consistent with the type used to instantiate the `read_state` template